src/components/Navbar/index.tsx

```diff
@@ -167,7 +181,16 @@ const Navbar: FC = () => {
 									onClick={() => navigateToPage(navItem.route)}
 									key={index}>
 									<Tooltip label={navItem.label} position="right">
-										<navItem.icon stroke={isActiveItem ? 1.4 : 1.2} size={'1.2rem'} />
+										{navItem.route === CORRELATION_ROUTE ? (
```

Comment:
active route can be handled with the predefined functions (same as other routes are handled)